### DISCUSSION TITLE
What models are best for very small defects on relatively smooth surface?

### DISCUSSION THREAD

**Discussion by haimat** [open-edge-platform/anomalib#2277] [General]
I have images like this:

![image](https://github.com/user-attachments/assets/64aa1769-a520-4722-bc58-762b11513eb3)

As you can see its quite a smooth, even, one-colored surface with small (white) dots on it.
What is the best model in Anomalib to deal with such defects/anomalies?

**Comment by alexriedel1** [comment]
You should try:
-Patchcore
-high image resolution
-tiling
-maybe adding sharpness increasing pre-processing

Depending on the variance of the defects you could also try "classical" methods like thresholding etc

**Comment by abc-125** [comment]
EfficientAD might also work, it has "tiling" by definition and it extracts features which imitate features from higher resolution.
